tests/shell/environment/configuration.rs

```diff
@@ -140,3 +140,35 @@ fn removes_config_values() {
         assert!(file_contents(&file).is_empty());
     })
 }
+
+#[test]
+fn updates_config_even_if_config_changed_externally() {
+    Playground::setup("updates_config_when_changed", |dirs, nu| {
+        let file = AbsolutePath::new(dirs.test().join("config.toml"));
+
+        nu.with_config(&file);
```

Comment:
@andrasio - all the configs are managed by the test harness now, right?

Is there a way to test changing from one config file to another?